tfe/data_source_workspace.go

```diff
@@ -193,6 +185,27 @@ func dataSourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("trigger_prefixes", workspace.TriggerPrefixes)
 	d.Set("working_directory", workspace.WorkingDirectory)
 
+	// Set remote_state_consumer_ids if global_remote_state is false
+	globalRemoteState := workspace.GlobalRemoteState
+	if globalRemoteState {
+		if err := d.Set("remote_state_consumer_ids", []string{}); err != nil {
+			return err
+		}
+	} else {
+		legacyGlobalState, remoteStateConsumerIDs, err := readWorkspaceStateConsumers(workspace.ID, tfeClient)
```

Comment:
Really minor cleanup suggestion: since `readWorkspaceStateConsumers()` returns a bool in a tuple, and if that value is true we set `globalRemoteState = true`, couldn't we just assign that function's return value to `globalRemoteState` instead?

Reply:
I didn't want to do that until I determined whether err != nil

Reply:
I kind of like it the way it is because it names the return value as a legacy compatibility issue